src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -8651,46 +8660,11 @@ namespace HVACVariableRefrigerantFlow {
         C2Tcond = DXCoil(HeatCoilNum).C2Tc;
         C3Tcond = DXCoil(HeatCoilNum).C3Tc;
 
-        // Rated air flow rate for the coil
-        if ((!VRF(VRFNum).HeatRecoveryUsed && CoolingLoad(VRFNum)) ||
-            (VRF(VRFNum).HeatRecoveryUsed && TerminalUnitList(TUListIndex).HRCoolRequest(IndexToTUInTUList))) {
-            // VRF terminal unit is on cooling mode
-            CompOnMassFlow = DXCoil(CoolCoilNum).RatedAirMassFlowRate(Mode);
-        } else if ((!VRF(VRFNum).HeatRecoveryUsed && HeatingLoad(VRFNum)) ||
-                   (VRF(VRFNum).HeatRecoveryUsed && TerminalUnitList(TUListIndex).HRHeatRequest(IndexToTUInTUList))) {
-            // VRF terminal unit is on heating mode
-            CompOnMassFlow = DXCoil(HeatCoilNum).RatedAirMassFlowRate(Mode);
-        }
-
-        // Set inlet air mass flow rate based on PLR and compressor on/off air flow rates
-        SetAverageAirFlow(VRFTUNum, 1.0, temp);
         VRFInletNode = this->VRFTUInletNodeNum;
         T_TU_in = Node(VRFInletNode).Temp;
         W_TU_in = Node(VRFInletNode).HumRat;
-        T_coil_in = T_TU_in;
-        W_coil_in = W_TU_in;
-
-        // Simulation the OAMixer if there is any
-        if (this->OAMixerUsed) {
-            SimOAMixer(this->OAMixerName, false, this->OAMixerIndex);
-
-            OAMixerNum = UtilityRoutines::FindItemInList(this->OAMixerName, OAMixer);
-            OAMixNode = OAMixer(OAMixerNum).MixNode;
-            T_coil_in = Node(OAMixNode).Temp;
-            W_coil_in = Node(OAMixNode).HumRat;
-        }
-        // Simulate the blow-through fan if there is any
-        if (this->FanPlace == BlowThru) {
-            if (this->fanType_Num == DataHVACGlobals::FanType_SystemModelObject) {
-                HVACFan::fanObjs[this->FanIndex]->simulate(1.0 / temp, ZoneCompTurnFansOn, ZoneCompTurnFansOff, _);
-                FanOutletNode = HVACFan::fanObjs[this->FanIndex]->outletNodeNum;
-            } else {
-                Fans::SimulateFanComponents("", false, this->FanIndex, FanSpeedRatio, ZoneCompTurnFansOn, ZoneCompTurnFansOff);
-                FanOutletNode = Fans::Fan(this->FanIndex).OutletNodeNum;
-            }
-            T_coil_in = Node(FanOutletNode).Temp;
-            W_coil_in = Node(FanOutletNode).HumRat;
-        }
+        T_coil_in = this->coilInNodeT;
+        W_coil_in = this->coilInNodeW;
```

Comment:
Change here is that coil inlet temps are now used so it doesn't matter if blow thru fan or OA mixer is used. Gets rid of a lot of unnecessary code.

Reply:
:+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: